core/case_admin/management/commands/seed_data.py

```diff
@@ -0,0 +1,616 @@
+                "email" : "jess@example.com",
+                "password" : "test123456789!"
+            }, 
+            {
+                "id" : 3, 
+                "firstName" : "Bobby" , 
+                "lastName" : "B", 
+                "email" : "bobby@example.com",
+                "password" : "test123456789!"
+            }, 
+            {
+                "id" : 4, 
+                "firstName" : "Sam" , 
+                "lastName" : "S", 
+                "email" : "sam@example.com",
+                "password" : "test123456789!"
+            }
+            , 
+            {
+                "id" : 5, 
+                "firstName" : "Sally" , 
+                "lastName" : "S", 
+                "email" : "sally@example.com",
+                "password" : "test123456789!"
+            }
+        ]
+
+        if action == "create":
+            for test_user in test_users:
+                try:
+                    user = User.objects.get(email=test_user["email"])
+                    print(test_user["email"] + " User has already been created")
+                except ObjectDoesNotExist:
+                    user = User.objects.create(email=test_user["email"])
+                    user.first_name = test_user["firstName"]
+                    user.last_name = test_user["lastName"]
+                    user.is_active = True 
+                    user.university = "UWA"
+                    user.degree_commencement_year = 2019
+                    user.set_password(test_user["password"]) 
+                    user.save()
+                    print("Created user " + test_user["email"] )
+
+        if action == "delete":
+            for test_user in test_users:
+                try:
+                    user = User.objects.get(email=test_user["email"]).delete()
+                    print(test_user["email"] + " User has been deleted")
+                except:
+                    print("Could not find or delete user " + test_user["email"])
+        
+        
+        # test questions
+        test_questions = [
+            {
+                "id":1,
+                "body": "test_What should you tell the patient?"
+            },
+            {
+                "id":2,
+                "body": "test_What dosage should you provide?"
+            },
+            {
+                "id":3,
+                "body": "test_Who should you refer this patient to?"
+            },
+        ]
+
+        if action == "create":
+            for test_question in test_questions:
+                try:
+                    question = Question.objects.get(body=test_question["body"])
+                    print("Question: '" + test_question["body"] + "' has already been created")
+                except ObjectDoesNotExist:
+                    question = Question.objects.create(body=test_question["body"])
+                    print("Question: '" + test_question["body"] + "' has been created")
+        
+        if action == "delete":
```

Comment:
For each `action == "delete"`, you can use the model to delete it, i.e. `CaseStudy.objects.all().delete()`. This means you can remove all the instances of `action == "delete"` and replace it with 1 single if statement

Reply:
Done :)